### ISSUE TITLE
Link a crash to a pull request and link it

### ISSUE THREAD

**Issue by BasThomas** [xing/crep#39]
Investigate if it would be possible / how foolproof it is to link a crash to a pull request that might have introduced it.

This is something that should probably be extracted and is not part of `crep` itself, but it would be really cool to have regardless.

This would comment on a pull request with a comment saying something like the following:

```
`crep` has detected that this pull request may have introduced a crash. You can find it [here](url-to-crashlog).
```

And then a 👍 or 👎 option to tell the bot if it behaved (read: is correct) or not (read: is incorrect).

**Comment by JoeSSS** [comment]
Don't think that people will often look into closed PR, maybe add options to create issues or send emails or messages in messenger like Slack and Co. (the message could point to the PR and the person who created it)

**Comment by BasThomas** [comment]
> Don't think that people will often look into closed PR

When a pull request is merged or closed, anyone subscribed will not be unsubscribed. That means that if we add a comment in a closed pull request, those subscribed will be notified as normal.
... and we can also explicitly mention people to trigger notifications if needed.

> maybe add options to create issues or send emails or messages in messenger like Slack and Co.

Definitely! But that would belong in a few separate issues I think 😉

**Comment by q231950** [comment]
@BasThomas I think it is quite a hard problem to solve. We could allow something like a _crash resolver_ that can feed information back to **Crep** upon some info from **Crep** so it knows which pr might have been the culprit. To me this feels like far beyond the current focus of **Crep**. 

**Crep** --- _line of code_ --> **Crash Resolver** --- _Pull Request_ --> **Crep**

It would need a separate proof of concept app, I guess. Think of an app that knows a line of code and a repository url and is supposed to figure out the relation of that line of code and the crash - I think it's difficult. Maybe you were thinking in another direction?

**Comment by BasThomas** [comment]
Yeah, definitely out of scope for Crep itself, which is why I labeled it that :)
